ibm/service/scc/resource_ibm_scc_account_settings.go

```diff
@@ -28,91 +29,299 @@ func ResourceIBMSccAccountSettings() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"location_id": {
 				Type:         schema.TypeString,
-				Required:     true,
+				Optional:     true,
 				ValidateFunc: validate.InvokeValidator("ibm_scc_account_settings", "location_id"),
 				Description:  "The programatic ID of the location that you want to work in.",
+				Deprecated:   "The attribute location_id will soon be deprecated. Please use location instead. See https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/scc_account_settings for details",
+			},
+			"location": &schema.Schema{
+				Type:          schema.TypeList,
+				MaxItems:      1,
+				Optional:      true, // Made this Required to avoid drift
+				ConflictsWith: []string{"location_id"},
+				Description:   "Location Settings.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"location_id": &schema.Schema{
+							Type:         schema.TypeString,
+							Required:     true,
+							Description:  "The programatic ID of the location that you want to work in.",
+							ValidateFunc: validate.InvokeValidator("ibm_scc_account_settings", "location_id"),
+						},
+					},
+				},
+			},
+			"event_notifications": &schema.Schema{
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Required:    true, // Made this Required to avoid drift during testing
```

Comment:
Still this will be breaking changes to the exisitng users...since it marked as required